modules/tinymce/CHANGELOG.md

```diff
@@ -101,6 +117,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed the `editor.settings` property as it's been replaced by the new Options API #TINY-8236
 - Removed the `shortEnded` and `fixed` properties on `tinymce.html.Node` class #TINY-8205
 - Removed the `mceInsertRawHTML` command #TINY-8214
+- Removed the undocumented `editor.editorCommands.hasCustomCommand` API #TINY-7829
```

Comment:
So, funny story, we are about to start using this API in #7569 to automatically disable table features not supported by RTC. It wasn't our preferred option but we ran out of time to do it a nicer way.

- [ ] please keep `hasCustomCommand` for this release - or maybe we should rename it `hasCommand`?

Reply:
Yeah I saw that. It could be replaced with queryCommandSupported since after this change it will all be custom commands. So the hasCustomCommand is now the same function as queryCommandSupported so it made no sense to keep it unless we want to remove the queryCommandSupported and have a hasCommand instead in the editorCommands but then it would mismatch with the editor.queryCommandSupported api that is documented and most likely used by integrators.

Reply:
Oh! yes I'm happy to swap the other branch to `queryCommandSupported`.